src/core/include/openvino/op/util/detection_output_base.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+
+namespace ov {
+namespace op {
+namespace util {
+/// \brief DetectionOutputBase basic class for DetectionOutput v0 and v8
+class OPENVINO_API DetectionOutputBase : public Op {
+public:
+    struct AttributesBase {
```

Comment:
Attributes?

Reply:
I would have Base in suffix because it is not attributes struct for the main classes